src/xlsx/xlsx_writer.ts

```diff
@@ -138,7 +139,29 @@ function createWorksheets(data: ExcelWorkbookData, construct: XLSXStructure): XL
           )
         );
       }
+    }
+
+    const images = sheet.images;
+    for (const image of images) {
+      const xlsxImageId = convertImageId(image.id);
+      const imageRelId = addRelsToFile(
+        construct.relsFiles,
+        `xl/drawings/_rels/drawing${sheetIndex}.xml.rels`,
+        {
+          target: `../media/image${xlsxImageId}`,
+          type: XLSX_RELATION_TYPE.image,
+        }
+      );
+      drawingRelIds.push(imageRelId);
+      files.push({
+        path: `xl/media/image${xlsxImageId}`,
+        content: image.data.path, // `file.content` stores relative path to avoid unnecessary data transfer
+        contentType: CONTENT_TYPES.image,
```

Comment:
Seeing what we do in https://github.com/odoo/o-spreadsheet/pull/1957, I'm fairly confident we should separate the images from the rest of the flow. It will also make it easier to document.